### PR TITLE
Update async sample code on the concurrency page #100

### LINKED ISSUE #100: Update async sample code on the concurrency page
# Tasks

- [x] Add "Use fs.promises" as first element
  - [x] Bad: fs.readFileSync()
  - [x] Good: fs.promises.readfile()

Estimated Time: 5min

# Writing the Issue

- [x] Write each task for this issue using checkboxes
- [x] Write all verbal instructions as tasks
- [x] Add labels and assignees

# Before Working on the Issue

- [x] Assign yourself
- [x] Share your screen on Discord
- [x] Write which issue you are starting on Slack
- [x] Git: Fetch the latest main
- [x] Git: Create a branch with the name of the issue

# Pre-PR Checks

- [x] Insure that changes include only what is necessary for this PR
- [x] Implement necessary Unit tests
- [x] Implement necessary E2E tests
- [x] Perform functionality checks

### PR DESCRIPTION
**Pull request by gottu91-code**
closes #100

# For the Submitter

## Tasks

- [x] Make sure "closes (issue number)" is written close to the top
- [x] Attach the enhancement label for update tasks
- [x] Make sure CI completes successfully
- [x] Add reviewers
- [x] On Slack, mention the PR reviewer, write your thoughts on the issue, and inform them that the PR has been created

# For the Reviewer

## Before Reviewing

- [ ] Write which PR you are reviewing on Slack

## Checks

- [ ] Each task for the related issue are written and in checkboxes
- [ ] The branch name is close to the issue name
- [ ] CI completes successfully
- [ ] Changes include only what is necessary for this PR
- [ ] Necessary unit tests are implemented
- [ ] Necessary E2E tests are implemented
- [ ] Perform functionality checks
- [ ] If there are any problems, request revisions from the PR submitter and mention them on Slack

## After Reviewing

- [ ] Leave an approval comment (e.g. LGTM)
- [ ] On Slack, mention the PR submitter and a project admin, write which PR you have reviewed, and ask them to merge the PR

### CODE REVIEW

File: docs/30-clean-code/60-concurrency.md [hunk 1]
```diff
@@ -3,6 +3,16 @@ title: Concurrency
 description: This is a guide to producing readable, reusable, and refactorable software for TypeScript.
 ---
 
+## Add "Use fs.promises" as first element
+
+```ts::Bad
+fs.readFileSync()
```
Comment (sinproject-iwasaki) [inline docs/30-clean-code/60-concurrency.md:9]:
```
import fs from 'fs

const file = fs.readFileSync()
```

File: docs/30-clean-code/60-concurrency.md [hunk 2]
```diff
@@ -3,6 +3,16 @@ title: Concurrency
...
+```
+
+```ts::Good
+fs.promises.readfile()
```
Comment (sinproject-iwasaki) [inline docs/30-clean-code/60-concurrency.md:13]:
```
import fs from 'fs

const file = await fs.readFileSync()
```

**Review by sinproject-iwasaki — CHANGES_REQUESTED** [review]
- [ ] 少しだけ調整依頼を記載しました。

**Comment by gottu91-code** [comment]
調整を行いました。
fs.promises.readfile()は
fs.promises.readFile()として記載しました。

**Pull request merged**